bench_method.go

```diff
@@ -63,19 +64,34 @@ func (m benchmarkMethod) call(t transport.Transport) (time.Duration, error) {
 	return duration, err
 }
 
+func hostPortBalancer(hostPorts []string) func(i int) string {
+	startOffset := rand.Intn(len(hostPorts))
+	return func(i int) string {
+		offset := (startOffset + i) % len(hostPorts)
```

Comment:
worth pulling `len(hostPorts)` into a variable? It's not going to change between calls.